app/views/queries/_sql_form_column.html.erb

```diff
@@ -11,6 +11,7 @@
         <ul>
           <li>Os nomes de variáveis podem conter caracteres de a-z, 0-9 e _</li>
           <li>Para utilizar uma variável na consulta utilize a notação <b>:nome_da_variavel</b> </li>
+          <li>Caso queira usar uma variável de data e formatá-la por extenso, identifique-a como ¨data¨</li>
```

Comment:
Se for fazer as alterações que sugeri, remova essa linha.

Se não fizer, é necessário que o comportamento seja replicado pra notificações, já que a consulta serve pra ambos